typescript-api/scripts/generate-types.ts

```diff
@@ -36,6 +36,7 @@ async function main() {
             "--dev-service",
             "--wasm-execution=interpreted-i-know-what-i-do",
             "--rpc-port=9933",
+            "--unsafe-force-node-key-generation"
```

Comment:
Is this new CLI option added? What does it do?

Reply:
it allows to generate automatically a node-key. Before this was done by default if you did not pass a key, now you need to explicitely say it